src/target/cortexm.c

```diff
@@ -565,6 +565,9 @@ bool cortexm_probe(adiv5_access_port_s *ap)
 	if (conn_reset)
 		target_mem32_write32(target, CORTEXM_DEMCR, 0);
 
+	DEBUG_TARGET("%s: Designer 0x%03x, selecting Part ID 0x%04x, AP Part ID: 0x%04x\n", __func__, target->designer_code,
```

Comment:
We still maintain that the designer code is not useful here, bloating the string needlessly - in the case of an unrecognised part, the designer gets printed in the below DEBUG_ERROR; and otherwise, the list of probe routines tried is printed when INFO is enabled and which gives which designer code path is taken. The designer codes are (unlike the part number) displayed above in the output as well, which helps.

"selecting" is going to be real odd in the output to a user, perhaps "Examining"?

Reply:
Applied.
The two significant numbers may differ for DPv2 but may match for conforming DPv1 implementations. It made more sense in STM32-specific code for the confused DPv2 targets. Keeping until logs seen.